scripts/crossgen/crossgen_roslyn.cmd

```diff
@@ -1,15 +1,15 @@
+@echo off
+
```

Comment:
That's what I'm fixing with this re-order... The `@echo off` was originally below the comments, causing exactly what you described. It no longer appears that way.